pkgs/by-name/ne/neovide/package.nix

```diff
@@ -94,9 +94,14 @@ rustPlatform.buildRustPackage.override { stdenv = clangStdenv; } rec {
 
       wrapProgram $out/bin/neovide \
         --prefix LD_LIBRARY_PATH : ${libPath}
+        --prefix PATH : ${lib.makeBinPath [ neovim python3 ]}
```

Comment:
Neovide has no dependency to python3, please remove.
Neovide has a dependency to neovim, but we should rely on the user's environment. I think you increase the closure size of this package with no need. Further more, if the user installed in his environment, it would be ignored.

Reply:
I've removed them. I only put them in because it was not working for @matteo-pacini, presumably because there it was an environment with no neovim install.

Reply:
Specifically, [this](https://github.com/NixOS/nixpkgs/pull/319061#pullrequestreview-2110944294):

> The app does not run properly on aarch64-darwin - I would be expect it to be wrapped so neovim is added to its PATH?